app/helpers/breadcrumb_helper.rb

```diff
@@ -27,36 +27,23 @@
 #++
 
 module BreadcrumbHelper
-  def full_breadcrumb
-    breadcrumb_list(*breadcrumb_paths)
-  end
-
-  def breadcrumb(*args)
-    elements = args.flatten
-    elements.any? ? content_tag('p', (args.join(' &#187; ') + ' &#187; ').html_safe, class: 'op-breadcrumb') : nil
-  end
-
-  def breadcrumb_list(*args)
-    elements = args.flatten
-    breadcrumb_elements = [content_tag(:li,
-                                       elements.shift.to_s,
-                                       class: 'first-breadcrumb-element')]
+  def full_breadcrumbs
+    items = breadcrumb_paths.compact
+    render(Primer::Beta::Breadcrumbs.new(data: { 'test-selector': "op-breadcrumb" })) do |breadcrumbs|
+      items.each_with_index do |item, index|
+        item = anchor_string_to_object(item) if item.is_a?(String) && item.start_with?("\u003c")
 
-    breadcrumb_elements += elements.map do |element|
-      if element
-        content_tag(:li,
-                    h(element.to_s),
-                    class: "icon4 icon-small icon-arrow-right5")
+        if item.is_a?(Hash)
+          breadcrumbs.with_item(href: item[:href], classes: index == 0 ? "first-breadcrumb-element" : nil) { item[:text] }
+        else
+          breadcrumbs.with_item(href: "#", classes: index == 0 ? "first-breadcrumb-element" : nil) { item }
```

Comment:
Q: Should we render these are links or normal text? Primer design doesn't seem to specify but I'm curious if this came up before 🤔 

The focus state seems misleading- and perhaps just raw text would be better
![Screenshot 2023-11-21 at 6 01 08 PM](https://github.com/opf/openproject/assets/17295175/eb254f05-81d7-4246-854f-738e0a038070)

Reply:
I just doubled checked with Primer, and they are saying in their definition of the breadcrumbs that the last element should be a plain text. So I think we should stick to that as well.

<img width="360" alt="Bildschirmfoto 2023-11-22 um 14 16 41" src="https://github.com/opf/openproject/assets/7457313/ccaf8d0f-896d-41cc-b1cf-294b570327a7">

Reply:
mhh.. again its really hard to understand Primer's intentions here. 🤷‍♀️ The override is fine for me for now. We should however keep an eye open to changes on the breadcrumb component in the future.